api/tasks/tasks.methods.tests.js

```diff
@@ -23,15 +23,15 @@ if (Meteor.isServer) {
       });
 
       it('can delete owned task', () => {
-        Meteor.call('removeTask', { taskId });
+        await Meteor.callAsync('removeTask', { taskId });
 
         assert.equal(Tasks.find().count(), 0);
       });
 
       it("can't delete task if not authenticated", () => {
```

Comment:
```suggestion
      it("can't delete task if not authenticated", async () => {
```

Could the codemod add the missing async on the context of the function where the await is added?

Reply:
#18, sorry for reverting back very late, I have updated the codemod accordingly.